views/faq.handlebars

```diff
@@ -270,6 +270,29 @@
               "makerspace" items to locate the ones that can use your contribution!
             </p>
           </dd>
+          <dt class="btn-link">
+            <h5 class="i18n" data-i18n="ftm-faq-sites-ftm-maker-partners">Who are #findthemasks Makers partners?</h5>
+          </dt>
+          <dd class="collapse">
+            <p class="i18n" data-i18n="ftm-faq-sites-ftm-maker-partners-answer-1">
```

Comment:
you will need to update to `data-i18n="[html]ftm-faq-sites-ftm-maker-partners-answer-1"` when you add the links in.

Reply:
the [html] will trigger the country-specific JS replacement??
That is SO cool

Reply:
Okay, so [html] tells the code that there is HTML here! and then retrieves from somewhere else in the code (no idea yet, probably another file?) which tells it what to do with HTML stuff????????